data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/TopicConfig.java

```diff
@@ -92,6 +93,9 @@ public class TopicConfig {
     @JsonProperty("max_record_fetch_time")
     private Duration maxRecordFetchTime = DEFAULT_MAX_RECORD_FETCH_TIME;
 
+    @JsonProperty("max_partition_fetch_bytes")
+    private Integer maxPartitionFetchBytes = DEFAULT_MAX_PARTITION_FETCH_BYTES;
```

Comment:
We should make use of Data Prepper's `ByteCount` type. See #3099 for an example of using this in the CloudWatch logs config.